datahub-web-react/cypress/integration/Login.spec.ts

```diff
@@ -1,40 +0,0 @@
-import { createLoginUsers } from '../../src/graphql-mock/fixtures/user';
-import { makeServer } from '../../src/graphql-mock/server';
-import { login, logout } from '../helper/authHelper';
-
-describe('Login', () => {
-    let server;
-
-    beforeEach(() => {
-        server = makeServer('test');
-        createLoginUsers(server);
-    });
-
-    afterEach(() => {
-        server.shutdown();
-    });
-
-    describe('given the login page is loaded', () => {
-        describe('when logging in with incorrect credentials', () => {
-            it('then the login should fail and the toast notification should be briefly displayed', () => {
```

Comment:
do we have these tests in our smoke tests project then?

Reply:
we have test for login